content/applications/general/voip/voip_widget.rst

```diff
@@ -0,0 +1,110 @@
+
+.. note::
+   The :abbr:`VoIP (Voice over Internet Protocol)` number is the one provided by Axivox. It can be
+   accessed by navigating to `https://manage.axivox.com/ <https://manage.axivox.com/>`_. After
+   logging into the portal, go to :menuselection:`Users --> Outgoing number` (column).
+
+.. image:: voip_widget/call.png
+   :align: center
+   :alt: VoIP call in Odoo.
+
+Troubleshooting
+===============
+
+.. tip::
+   If a *Missing Parameter* error message appears in the Odoo *VoIP* widget, refresh the Odoo
+   window, and try again.
+
+   .. image:: voip_widget/missing-parameter.png
+      :align: center
+      :alt: "Missing Parameter" error message in the Odoo softphone.
+
+.. tip::
+   If an *Incorrect Number* error message appears in the Odoo *VoIP* widget, make sure to use the
+   international format, leading with the :guilabel:`plus (+)` sign, followed by the international
+   country code.
+
+   (E.g., +16506913277, where `+1` is the international prefix for the United States.)
+
+   .. image:: voip_widget/incorrect-number.png
+      :align: center
+      :alt: "Incorrect Number" error message in the Odoo softphone.
+
+Tabs
+====
+
+In all, there are three tabs (Recent, Next Activities, Contacts) present in the *VoIP* widget used
+for managing calls and activities day-to-day in Odoo.
+
+Recent
+------
+
+Under the :guilabel:`Recent` tab of the *VoIP* widget, the call history for the user is available.
+This includes incoming and outgoing calls. Any number can be clicked to begin a call.
+
+Next activities
+---------------
+
+Under the :guilabel:`Next Activities` tab of the *VoIP* widget, a user can see any activities
+assigned to them, and are due to be completed for the day.
+
+Click an activity from this tab to perform any actions including: Sending an email, accessing their
+contact, scheduling another activity, or accessing a linked record (such as a Sales Order,
+Lead/Opportunity, or Project Task).
+
+The user can also mark the activity as complete, edit the details of the activity, or cancel it.
+
+To call the customer related to a scheduled activity, click the :guilabel:`📞 (phone)` icon, or
+click the :guilabel:`⌨️ (keyboard)` icon to dial another number for the customer.
+
+.. image:: voip_widget/activity-widget.png
+   :align: center
+   :alt: Activity control center on the VoIP widget.
+
+Some other icons appear in the *VoIP* widget, categorized by two sections: :guilabel:`Document` and
+:guilabel:`Activity`.
+
+Under the :guilabel:`Document` section, from right to left:
+
+- :guilabel:`➣ (paper airplane)` icon: sends an email
+- :guilabel:`👤 (person icon)` icon: redirects to the contact card
+- :guilabel:`📄 (document)` icon: redirects to the attached record in Odoo
+- :guilabel:`🕓 (clock)` icon: schedule an activity
+
+Under the :guilabel:`Activity` section, from left to right:
+
+- :guilabel:`✔️ (checkmark)` icon: mark activity as document
```

Comment:
mark activity as "document" or "done"? just want to make sure we're delivering the proper information

Reply:
Done